client/src/app/pages/applications/components/application-form/application-form.tsx

```diff
@@ -683,3 +583,95 @@ export const ApplicationForm: React.FC<ApplicationFormProps> = ({
     </Form>
   );
 };
+
+const useApplicationFormData = ({
+  onActionSuccess = () => {},
+  onActionFail = () => {},
+}: {
+  onActionSuccess?: () => void;
+  onActionFail?: () => void;
+}) => {
+  const { t } = useTranslation();
+  const { pushNotification } = React.useContext(NotificationsContext);
+
+  // Fetch data
+  const { tagCategories } = useFetchTagCategories();
+  const tags = useMemo(
+    () => tagCategories.flatMap((tc) => tc.tags).filter(Boolean) as Tag[],
```

Comment:
Is there any way to let typescript in on this .filter(Boolean) magic?

Reply:
Ohhhh, just found this nugget: https://www.karltarvas.com/typescript-array-filter-boolean.html

Dropped a file in client/src/types with the extra definition and it's working!  :grin:

Reply:
Oh my..... that is a beautiful thing!